pages/common/pnpx.md

```diff
@@ -0,0 +1,16 @@
+# pnpx
+
+> Directly execute binaries from npm packages, using `pnpm` instead of `npm`.
+> More information: <https://pnpm.io/pnpx-cli>.
+
+- Execute the binary from a given npm module:
+
+`pnpx {{module_name}}`
+
+- Execute a specific binary from a given npm module, in case the module has multiple binaries:
+
+`pnpx --package {{package_name}} {{module_name}}`
```

Comment:
This is confusing. The language in the example talks about binaries, but then it mentions a package name in the example? :thinking:

Reply:
@sbrl We have the same example in the `npx` page https://github.com/tldr-pages/tldr/blob/main/pages/common/npx.md

Reply:
Ah, ok. I must not be as familiar with npx as I thought :woman_shrugging: